lib/profiles/TR/WG-NOTE.js

```diff
@@ -11,7 +11,7 @@ exports.config = {
 
 const base = require("../base");
 
-exports.rules = base.extendWithInserts({
+const rules = base.extendWithInserts({
```

Comment:
`base.RemoveRules` does reassign the first argument so if rules is a constant, there will be an exception.
```suggestion
var rules = base.extendWithInserts({
```